index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Sandro's portfolio</title>
+</head>
+<body>
+  <div class="general">
+    <header>
+      <div class="mobile">
+        <div id="time">9:41</div>
+        <div id="icons">
+          <img src="./Images/Cellular Connection.png" alt="signal">
+          <img src="./Images/Wifi.png" alt="wifi">
+         <img src="./Images/Battery.png" alt="batery">
+       </div>
+     </div>
+
+     <div class="menu">
+        <p>Jonh Doe</p>
+        <a href="#"><img src="./Images/Union.png" alt="menu"></a>
```

Comment:
- [ ] It is required to wrap your hamburger inside the `<nav>` tag for semantic reason. Kindly wrap your hamburger inside a `<nav>` tag.